src/components/action-button.js

```diff
@@ -12,14 +12,13 @@ export const runActions = async actionList => {
 export default function ActionButton({ layout, button, Theme, app, context, senseNavigation, element }) {
   const { style, qStateName } = layout;
   const disabled = layout.useEnabledCondition && layout.enabledCondition === 0;
-  const formattedStyles = styleFormatter.getStyles({ style, disabled, Theme, element });
+  const formattedStyles = styleFormatter.getStyles({ style, disabled, Theme, element, button });
   button.setAttribute('style', formattedStyles);
```

Comment:
I think we can move this into `getStyles` and make it a `setStyles`, more aligned with setFont

Reply:
Yes good idea. I will open a new PR for that. Will need to rewrite a lot of unit tests for that since we now check the return value in getStyles